### PR TITLE
Bugfix/258 header responsive

### PR DESCRIPTION
**Pull request by Ruby77**
## Description

Make the header responsive for different screen sizes. I did this by adding bootstrap classes for a burger menu. And made this trigger by using typescript (and styled the navbar in the burger menu). (Because there a still no designs for the header, this will be used for a starting point for whenever the design updates)

## Type of change

-   [x] Bug fix (non-breaking change which fixes an issue)
-   [ ] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
-   [ ] This change requires a documentation update

## Checklist

-   [x] My code follows the style guidelines of this project
-   [x] I have performed a self-review of my own code
-   [x] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have made corresponding changes to the documentation
-   [ ] I have added tests that prove my fix is effective or that my feature works
-   [x] I updated the changelog with an end-user readable description
-   [x] I assigned this pull request to the correct project board to update the sprint board

## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Make screen 767px wide. The links in the header should disappear and a burger menu should appear.

2. Click on the burger menu. The links should appear under the header, aligned to the left.

3. When you are logged in, you should be able to click on your account name. When clicking on your name, the logout text should appear under the current links.

## Link to issue

Closes: #258

### CODE REVIEW

**Comment by Brend-Smits** [comment]
@Niekvandam and @liekevdvoort, mind reviewing this PR?

**Review by liekevdvoort — DISMISSED** [review]

**Review by Niekvandam — DISMISSED** [review]

File: src/app/components/app-layout/app-layout.component.ts [hunk 1]
```diff
@@ -94,4 +95,12 @@ export class AppLayoutComponent implements OnInit {
   public onClickHeaderBetaText(): void {
     this.displayBetaBanner = true;
   }
+
+  /**
+   * Method which triggers when the user clicks on the burger menu in the header.
+   * Toggles the navbar visibility
+   */
+  public toggleNavbar() {
```
Comment (Brend-Smits) [inline src/app/components/app-layout/app-layout.component.ts:103]:
```suggestion
  public toggleNavbar(): void {
```

**Review by Brend-Smits — CHANGES_REQUESTED** [review]
Functional, nice! Thanks for the PR!

File: CHANGELOG.md [hunk 2]
```diff
@@ -14,6 +14,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Changed all file line endings from CRLF to LF and added the .gitattributes to enforce it - [#163](https://github.com/DigitalExcellence/dex-backend/issues/163)
 - Changed footer styling to be responsive on small screens - [#163] (https://github.com/DigitalExcellence/dex-frontend/issues/163)
```
Comment (Brend-Smits) [inline CHANGELOG.md:16]:
```suggestion
- Changed footer styling to be responsive on small screens - [#163](https://github.com/DigitalExcellence/dex-frontend/issues/163) 
```

**Review by Brend-Smits — CHANGES_REQUESTED** [review]

**Review by liekevdvoort — DISMISSED** [review]

**Review by Brend-Smits — APPROVED** [review]
You can merge this when you fix the conflicts

**Review by Brend-Smits — APPROVED** [review]

**Review by liekevdvoort — APPROVED** [review]

**Pull request merged**